aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/JobsServiceImpl.java

```diff
@@ -208,10 +208,14 @@ public String startDeepAnalysis(String applicationGuid, String nodeName, String
         if (moduleGenerationType != null && (moduleGenerationType != ModuleGenerationType.FULL_CONTENT)) {
             requestBuilder.moduleGenerationType(moduleGenerationType.toString());
         }
+        return startDeepAnalysis(requestBuilder.build());
+    }
 
-        log.info("Job Parameters: " + requestBuilder.build().toString());
+    @Override
+    public String startDeepAnalysis(ScanAndReScanApplicationJobRequest fastScanRequest) throws JobServiceException {
+        log.info("Job Parameters: " + fastScanRequest.toString());
```

Comment:
Maybe make this as a debug message, it's probably not worth putting the entire configuration of the fastscan object in the logs

Reply:
ok goes to debug level then

Reply:
Sorry didn't see your reply, "fine" is what would match "debug" (finer/finest being basically trace)